eos-updater/fetch.c

```diff
@@ -464,12 +464,13 @@ install_flags_for_action_flags (EuuFlatpakRemoteRefActionFlags action_flags)
       FLATPAK_INSTALL_FLAGS_NONE;
 }
 
-static FlatpakInstallFlags
+static FlatpakUpdateFlags
```

Comment:
I am surprised that the compiler did not catch this earlier! Is there a warning level we can use to get strongly typed enums?

Reply:
I was surprised too, really